rules/gandalf.js

```diff
@@ -0,0 +1,3 @@
+module.exports = {
+  "id-length": "off"
```

Comment:
id-length must be inside rules property - [example](https://github.com/SoftboxLab/gandalf-lint/blob/feature/new-rules/rules/gandalf.js)